code/game/machinery/camera/tracking.dm

```diff
@@ -161,6 +161,8 @@
 			return FALSE
 		if(istype(target.loc, /obj/effect/dummy))
 			return FALSE
+		if(ispulsedemon(M))
```

Comment:
it is also my opinion that they should be trackable if the floor tile's been pried off